assets/native_assets.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_ASSETS_NATIVE_ASSETS_H_
+#define FLUTTER_ASSETS_NATIVE_ASSETS_H_
+
+#include <memory>
+#include <vector>
+
+#include "flutter/assets/asset_manager.h"
+
+namespace flutter {
+
+class NativeAssetsManager {
```

Comment:
Can you add a short doc comment, possibly with some breadcrumbs to the relevant native assets documentation?